testing/src/Environment.php

```diff
@@ -44,7 +44,7 @@ public function start(string $rrCommand = null): void
 
         $this->output->write('Starting Temporal test server... ');
         $this->temporalServerProcess = new Process(
-            [$this->systemInfo->temporalServerExecutable, 7233, '--enable-time-skipping']
+            [$this->systemInfo->temporalServerExecutable, getenv('TEMPORAL_ADDRESS'), '--enable-time-skipping']
```

Comment:
Maybe we can add a default value here? Like this:

```php
getenv('TEMPORAL_ADDRESS') ?: '127.0.0.1:7233`
```

Or pass the variable into `Environment::create()` constructor and hide this "default logic" there. The same about trait and `WorkflowTestCase`. Let's add deafult value there.

Reply:
I added default value :)